packages/SwingSet/src/kernel/kernel.js

```diff
@@ -710,6 +710,7 @@ export default function buildKernel(
       // eslint-disable-next-line @jessie.js/no-nested-await
       await vatWarehouse.createDynamicVat(vatID);
     } catch (err) {
+      console.log('error during creation', err);
```

Comment:
make this one `error during createDynamicVat`, just to add some more context

Reply:
Done